relay-metrics/src/aggregator.rs

```diff
@@ -526,7 +540,7 @@ impl Aggregator {
             aggregator = &self.name,
         );
 
-        let mut buckets = HashMap::<ProjectKey, Vec<HashedBucket>>::new();
+        let mut buckets = HashMap::<ProjectKey, Vec<_>>::new();
```

Comment:
```suggestion
        let mut buckets = HashMap::<ProjectKey, Vec<Bucket>>::new();
```

i dont really see why we should have fewer annotations when it doesn't add verbosity, maybe im missing something

Reply:
It was annoying me, because if it wasn't specified in the first place I wouldn't have had to change it because the compiler would have correctly inferred it.

But I can actually completely get rid of the annotations, will do that instead.